pkg/core/state/notification_event.go

```diff
@@ -120,6 +130,9 @@ func (aer *AppExecResult) DecodeBinary(r *io.BinReader) {
 	aer.Stack = arr
 	r.ReadArray(&aer.Events)
 	aer.FaultException = r.ReadString()
+	if aer.VMState&0x80 != 0 {
```

Comment:
After we've used `aer.VMState` for decoding, we need to erase the trace of `0x80` from `aer.VMState`, it's a pure DB-level bit that must not be seen by external users.